File: scripts/install-frontend.py
import subprocess
import os

frontend_dir = "/vercel/share/v0-project/services/2016-roblox-main"
print(f"Installing dependencies in: {frontend_dir}")
result = subprocess.run(
    ["npm", "install"],
    cwd=frontend_dir,
    capture_output=True,
    text=True,
    timeout=180
)
print("STDOUT:", result.stdout)
print("STDERR:", result.stderr)
print("Return code:", result.returncode)

File: scripts/install-frontend.mjs
import { execSync } from 'child_process';
import path from 'path';

const frontendDir = path.join('/vercel/share/v0-project/services/2016-roblox-main');
console.log('Installing dependencies in:', frontendDir);
try {
  const result = execSync('npm install', { cwd: frontendDir, encoding: 'utf-8', timeout: 120000 });
  console.log(result);
} catch (err) {
  console.error('Error:', err.message);
  console.error(err.stdout);
  console.error(err.stderr);
}
